src/parsing/pyproject.ts

```diff
@@ -65,6 +67,41 @@ class PyprojectTOMLVisitor implements Visitor<TOMLNode> {
         }
     }
 
+    private potentiallyRegisterPixiDependency(node: TOMLTable | TOMLKeyValue): void {
+        if (this.pathStack[0] === 'tool' && this.pathStack[1] === 'pixi') {
+            const projectName: string | undefined = (() => {
+                if (
+                    (this.pathStack[2] as string) === 'dependencies' &&
+                    this.pathStack.length === 4 &&
+                    typeof this.pathStack[3] === 'string'
+                ) {
+                    // Basic dependencies
+                    return this.pathStack[3]
+                }
+                if (
+                    this.pathStack[2] === 'feature' &&
+                    this.pathStack[4] === 'dependencies' &&
+                    this.pathStack.length === 6 &&
+                    typeof this.pathStack[5] === 'string'
+                ) {
+                    // Dependency feature
+                    return this.pathStack[5]
+                }
+                return undefined
+            })()
+
+            if (projectName) {
+                this.dependencies.push([
+                    {
+                        name: projectName,
+                        type: 'ProjectName',
+                    },
+                    [node.loc.start.line - 1, node.loc.start.column, node.loc.end.line - 1, node.loc.end.column],
+                ])
+            }
```

Comment:
Two changes here:
1. let's detect `pypi-dependencies` instead of `dependencies`
2. let's avoid the self-calling function (IIFE) – a regular if/else with `let projectName` will be more readable

Reply:
Fixed by [e99d142](https://github.com/Twixes/pypi-assistant/pull/37/commits/e99d1427a4c7e11702add482949dcebe0ded6672)